applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -262,6 +270,19 @@ def __init__(self, model, custom_settings):
         ## Construct the linear solver
         self.linear_solver = linear_solver_factory.ConstructSolver(self.settings["linear_solver_settings"])
 
+        if not self.settings["turbulence_model"].IsEquivalentTo(KratosMultiphysics.Parameters("{}")):
+            # if not empty
+            if CheckIfApplicationsAvailable("RANSModellingApplication"):
+                import KratosMultiphysics.RANSModellingApplication as KratosRANS
+            else:
+                raise Exception("Please install/compile RANSModellingApplication to use turbulence_model properties")
+            from KratosMultiphysics.FluidDynamicsApplication.turbulence_model_configuration import CreateTurbulenceModel
+            self.turbulence_model_configuration = CreateTurbulenceModel(model, self.settings["turbulence_model"])
+            self.condition_name = self.turbulence_model_configuration.GetFluidVelocityPressureConditionName()
```

Comment:
What does this one do? I guess this will come in a future PR?

Reply:
Yes, this part at the moment does nothing, raises an exception since RANSModellingApplication is not present yet